mod-infra/src/main/scala/at/forsyte/apalache/infra/ExitCodes.scala

```diff
@@ -0,0 +1,57 @@
+package at.forsyte.apalache.infra
+
+import tlc2.output.EC
+
+/**
+ * Exit codes that are reported by Tool.
+ *
+ * We are using a small subset of exit codes. Although we maintain exit codes that are compatible with TLC, we believe
+ * that exit codes should not replace richer error explanations.
+ *
+ * TLC currently specifies exit codes
+ *   - 0 for success,
+ *   - 10–14 for (various kinds of) property violations,
+ *   - 75–77 for "failures" (e.g., properties that are tautologies, or primed variables in invariants), and
+ *   - 150–153 and 255 for syntax errors and system-level errors.
+ *
+ * In addition, Apalache uses exit code
+ *   - 120 for type-checking errors.
+ *
+ * @author
+ *   Igor Konnov, Thomas Pani
+ */
+object ExitCodes {
+  type TExitCode = Int
```

Comment:
What's gained by having a type alias for `Int`? And why not just call it `ExitCode`?

Reply:
> What's gained by having a type alias for `Int`?

a) I believe that type names are important in conveying meaning, e.g., a return type of `TExitCode` is more meaningful than `Int`.
b) Exit codes range from 0..255, they are definitely not `Int`s ;)

> And why not just call it `ExitCode`?

I want to use both the type and the object with unprefixed names. If Scala 2 allowed top-level type declarations (as does Scala 3), the type would anyhow live next to the `object` and be named differently to avoid a name clash.